Sources/Danger/GitLabDSL.swift

```diff
@@ -33,7 +33,7 @@ extension GitLab {
                 case createdAt = "created_at"
                 case description
                 case dueDate = "due_date"
-                case id // swiftlint:disable:this identifier_name
+                case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 36;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)